core/embed/trezorhal/trng.c

```diff
@@ -19,20 +19,20 @@
 
 #include STM32_HAL_H
 
-#include "rng.h"
+#include "trng.h"
 
 #pragma GCC optimize( \
     "no-stack-protector")  // applies to all functions in this file
 
-void rng_init(void) {
+void trng_init(void) {
   // enable TRNG peripheral clock
   // use the HAL version due to section 2.1.6 of STM32F42xx Errata sheet
   // "Delay after an RCC peripheral clock enabling"
   __HAL_RCC_RNG_CLK_ENABLE();
   RNG->CR = RNG_CR_RNGEN;  // enable TRNG
 }
 
-uint32_t rng_read(const uint32_t previous, const uint32_t compare_previous) {
+uint32_t trng_read(const uint32_t previous, const uint32_t compare_previous) {
```

Comment:
> The naming of these functions trng_read and trng_random32 is no more illuminating than it was before. 

The only reason why I renamed `rng_get` to `trng_random32` is I wanted it to be consistent with `drbg_random32`. I have no problem with reverting that back.

> It would be nice to get rid of that if we can so that we expose only one of these.
I would love to but I didn't wanted to modify the boardloader in any way.

Reply:
Done in eb89f674a9144a1c9e83f9fc23a4f27598c7fddd.